test/Microsoft.Azure.Devices.Edge.Test/helpers/Context.cs

```diff
@@ -97,8 +97,9 @@ public Context()
             this.RootCaKeys = GetAndValidateRootCaKeys();
             this.SetupTimeout = TimeSpan.FromMinutes(context.GetValue("setupTimeoutMinutes", 5));
             this.TeardownTimeout = TimeSpan.FromMinutes(context.GetValue("teardownTimeoutMinutes", 2));
-            this.TempSensorImage = Option.Maybe(Get("tempSensorImage"));
+            this.TempFilterFunc = Option.Maybe(Get("tempFilterFunc"));
```

Comment:
It pains me to make the variable longer, but this one should end in "Image" like the others (at least the context.json parameter, and probably the C# property as well. 😢 If we want to keep it shorter, I don't mind "tempFunctionImage".

Reply:
That's very reasonable. The member variable is now `TempFilterFuncImage` for the config JSON `tempFilterFuncImage`